### ISSUE TITLE
Integer overflow in ExtractHttpRanges

### ISSUE THREAD

**Issue by marko444** [ServiceStack/Issues#718]
Can you please check this exception.
```
Exception=System.OverflowException: Value was either too large or too small for an Int32.
   at System.Number.ParseInt32(String s, NumberStyles style, NumberFormatInfo info)
   at ServiceStack.HttpResultUtils.ExtractHttpRanges(String rangeHeader, Int64 contentLength, Int64& rangeStart, Int64& rangeEnd) in ...
```
Tnx!

**Comment by mythz** [comment]
Do you have a repro? Or can you provide the raw HTTP Request headers that’s causing this?

**Comment by marko444** [comment]
Issue happened in IE (but not in chrome, oddly). 
Scenario - seeking over a  4GB large video, retrieving content. SS code failed when calling .ExtractHttpRanges() on any of these values from range header:

> bytes=0-4203195464

> bytes=4200955904-4203195464

I have a little bit of mess currently with all the dll's so I just assumed it came from SS?

Regards!

**Comment by mythz** [comment]
Crap I thought I replied to this issue already, was just waiting for CI to finish publishing the package before closing the issue. I misread the original issue thought it was a StackOverflow Exception which I couldn't see how it was possible, but realized after it was just a range overflow exception.

Anyway I've resolved this issue and improved `ExtractHttpRanges` general handling which is now available from v5.8.1 that's [now available on MyGet](https://docs.servicestack.net/myget).
